TerminalDocs/command-line-arguments.md

```diff
@@ -8,91 +8,171 @@ ms.topic: how-to
+> [!NOTE]
+> If you built the Windows Terminal from the source code on [GitHub](https://github.com/microsoft/terminal), you can open that build using `wtd.exe` or `wtd`.
 
 ![Windows Terminal command line argument for split panes](./images/terminal-command-args.gif)
 
 ## Command line syntax
 
-The `wt` command line accepts two types of values: options and commands:
+The `wt` command line accepts two types of values: **options** and **commands**. **Options** are a list of flags and other parameters that can control the behavior of the `wt` command line as a whole. **Commands** provide the action, or list of actions separated by semicolons, that should be implemented. If no command is specified, then the command is assumed to be `new-tab` by default.
 
 ```bash
 wt [options] [command ; ]
 ```
 
-Options are a list of flags and other parameters that can control the behavior of the `wt` command line as a whole. Commands are a semicolon-delimited list of commands and arguments for those commands.
-
-If no command is specified, then the command is assumed to be `new-tab` by default. This allows for the following interpretations:
+To display a help message listing the available command line arguments, enter: `wt -h`, `wt --help`, or `wt -?`.
 
-| Command | Interpretation |
-| ------- | -------------- |
-| `wt cmd.exe` | `wt new-tab cmd.exe` |
-| `wt ; ; ;` | `wt new-tab ; new-tab ; new-tab ; new-tab` |
+## Options and commands
 
-## Command line options
+Below is the full list of supported commands and options for the `wt` command line.
 
 | Option | Description |
 | ------ | ----------- |
 | `--help`, `-h`, `-?`, `/?` | Displays the help message. |
 
-## Command line commands
+| Command | Parameters | Description |
+| ------- | ---------- | ----------- |
+| `new-tab` | `--profile, -p profile-name`, `--startingDirectory, -d starting-directory`, `commandline` | Creates a new tab. |
+| `split-pane` | `-H, --horizontal`, `-V, --vertical`, `--profile, -p profile-name`, `--startingDirectory, -d starting-directory`, `commandline` | Splits a new pane. |
+| `focus-tab` | `--target, -t tab-index` | Focuses on a specific tab. |
+
+## Command line argument examples
 
-### Create a new tab
+Commands may vary slightly depending on which command line you're using.
 
-This opens a new tab with the given customizations. On its first invocation, this also opens a new window. Subsequent `new-tab` commands will all open new tabs in the same window.
+### Open a new profile instance
+
+To open a new Terminal instance, in this case the command will open the profile named "Ubuntu-18.04", enter:
 
 ```bash
-new-tab [parameters]
+# Command Prompt
+wt -p "Ubuntu-18.04"
+
+# PowerShell
+wt -p "Ubuntu-18.04"
+
+# Linux distro
+cmd.exe /c "wt.exe" -p "Ubuntu-18.04"
+
+# Execution aliases do not work in WSL distributions. If you want to use wt.exe from a WSL command line, you can spawn it from CMD directly by running `cmd.exe`. The `/c` option tells CMD to terminate after running.
```

Comment:
Long comments like this don't wrap in the code blocks on the website. I recommend breaking them up into multiple lines